### ISSUE TITLE
Network status LED off

### ISSUE THREAD

**Issue by st0nec0ld** [heisath/wdmc2-kernel#12]
The right back panel network LED is off, but network works!  

![PXL_20220508_100321394](https://user-images.githubusercontent.com/21226229/167310478-a0561870-b87a-49da-ae4e-81213013582e.jpg)

The left is green blinking and shows activ communications.

Tested the version from https://wd.hides.su/fox_exe/WDMyCloud-Gen2/Debian/ the right status LED is fine.

**Comment by Heisath** [comment]
Hey thanks for reporting this. I never thought about the second LED, what extra information does one gain? 

I'd assume this is a difference in the kernel driver between linux kernel 4.x and 5.y but I am unsure. I have no active patches in that direction though.

**Comment by st0nec0ld** [comment]
It is called the "Link LED"

![image](https://user-images.githubusercontent.com/21226229/167460645-9e9e0e85-0922-4ef0-a9ec-4236e929ceb5.png)

it still works anyway but it is not normal ^^

**Comment by Heisath** [comment]
Yeah I am not sure why/where it breaks. I went all the way back to kernel 4.15 and the led was still not working. Maybe it is a debian/userspace problem.  If you find the cause I'd gladly fix it, but I don't think it is worth spending lots of time on.

**Comment by erpadmin** [comment]
following another project and recall seeing this:
https://github.com/gisab/WDMC-Ex2/tree/main/3.Kernel
see section "Use patch to fix LED beheavor"
unconfirmed but seems related

**Comment by Heisath** [comment]
Can you try if #21 fixes the problem?
